web/src/lib/graphql/Query/Query.js

```diff
@@ -23,14 +29,15 @@ const Query = ({ skeleton, children, ...others }) => {
           );
         }
 
-        return children({ loading, error, data });
```

Comment:
I do prefer this, but I think it'll be nice to pass `error` and `loading` as 2nd and 3rd parameters "just incase" a user needs it.

Reply:
The way it's written right now, the only way to get to that line is if `loading` and `error` are both undefined, so passing them down would be pointless! I really like the ability to declaratively create the skeleton, could we do the same for an error state and then keep with always just passing down the data? It would make it really easy to reason about the component if you were guaranteed to get valid data if it is being rendered.

Reply:
Ah, you're right! 🤦‍♂️

> could we do the same for an error state and then keep with always just passing down the data

Yup! I'll start working on that.